README.txt

```diff
@@ -8,7 +8,9 @@
 This mod adds a wooden trash can, and a dumpster to the game.  Right click it, put in your trash, and click the empty trash button.
 You can also throw things in the wooden trash can by pressing "q" or throwing them out of your inventory.
 
-Version: 0.2.2
+This is a "P" version, based on 0.2.2 enhanced with ability to run with MineClone2 game.
```

Comment:
Also good to mention Minetest Game in this case: `After version 0.2.3, this mod is compatible with Minetest Game and MineClone 2`.